clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -41,66 +54,208 @@ interface InputProps {
 type StringField = FieldHookConfig<string | undefined>;
 type StringArrayField = FieldHookConfig<string[] | undefined>;
 
+const Label = ({
+  title,
+  ...labelProps
+}: {
+  title: string;
```

Comment:
Making this an actual `children: ReactNode` could save us some work if/when we need build this out to match Chakra customizability.